src/main/java/com/bitquest/bitquest/InventoryEvents.java

```diff
@@ -59,17 +59,16 @@ public InventoryEvents(BitQuest plugin) {
         trades.add(new Trade(new ItemStack(Material.CHORUS_FLOWER,8),20000));
         trades.add(new Trade(new ItemStack(Material.DIAMOND,32),20000));//honestly needed more than 8
```

Comment:
I think more is good, but I would increase the price too.  If nothing else, diamonds are potentially one area where players could compete with the villager marketplace on goods, so I don't think we should price them out.